manim/utils/tex_file_writing.py

```diff
@@ -215,32 +215,33 @@ def convert_to_svg(dvi_file, extension, page=1):
         Path to generated SVG file.
     """
     result = dvi_file.replace(extension, ".svg")
-    result = Path(result).as_posix()
+    result = Path(result)
+    result_str = result.as_posix()
```

Comment:
I think this would not be required if you made `file_name` be a Path object in `svg_mobject.py` and hence would clear up some logic here by not having to track two seperate types `result (Path)` and `result_str (string)`

Reply:
I don't quite know an easy way to fix this. The parameters of the tex_compilation_command function provides a string so I'm a little confused on how I would be able to modify this code.

Reply:
Then do not add `Path` here, it is unnecessary